openvdb/util/Name.h

```diff
@@ -35,6 +37,77 @@ writeString(std::ostream& os, const Name& name)
     os.write(&name[0], size);
 }
 
+namespace util {
```

Comment:
I think I'd be tempted to put these in a new namespace for extra readability - openvdb::util::split() doesn't tell you anything about what it is you're actually splitting for example. Perhaps openvdb::string::split() would be clearer?

There are other non-util namespaces defined in this util sub-directory (such as openvdb::logging) so it wouldn't be breaking with precedence there.

Reply:
Happy to go with `openvdb::string` for now